packages/big-design/src/components/Table/HeaderCell/HeaderCell.tsx

```diff
@@ -133,7 +131,7 @@ export const DragIconHeaderCell: React.FC<DragIconCellProps> = memo(
     const actionsSize = useComponentSize(actionsRef);
 
     return (
-      <StyledTableHeaderCell
+      <StyledTableHeaderIcon
         ref={headerCellIconRef}
         stickyHeight={actionsSize.height}
         width={width}
```

Comment:
I believe we should be able to remove the `width` prop now!

Reply:
Good point, thought it was being used somewhere else in the file

Reply:
Would it be worth it also removing the width from the `DragIconCellProps` interface as well?

Reply:
That's amazing!